package.json

```diff
@@ -26,7 +26,7 @@
   },
   "homepage": "https://github.com/bigcommerce/checkout-js#readme",
   "dependencies": {
-    "@bigcommerce/checkout-sdk": "^1.124.1",
+    "@bigcommerce/checkout-sdk": "^1.125.0",
```

Comment:
Can you update `package-lock.json` as well?

Reply:
fixed